### PR TITLE
Lf 3271 remove unused code

### PR DESCRIPTION
**Pull request by Duncan-Brain**
**Description**

This continues the work of #1920. It is a whopping 14,305 deletions and 94 files changed. Some of the feature deletion may extend beyond the other PR files touched but tries to keep the same scope.

The commits try to focus on one single feature at a time. This should allow the ability to git cherry pick and revert any feature that is wanted in the future. There are a few commits that apply to a few features.

The commits remove the following:
- unused crypto keypair utility
- export all farm data feature (GDPR?)
- Insight - People Fed
- Insight - Erosion
- Insight - Water Balance & Scheduler
- Insight - Nitrogen balance & Scheduler
- Insight - shared components between Water Balance and Nitrogen Balance
- Sales Summary
- unused getAuth0Url utility
- Shift and shift task feature
- continue removing shift and shift task feature
- unused userFarm enum
- unused Job - compress image
- unused URLs in apiConfig
- Insight - helper function
- non-existent routes
- merge integration

Things I did not keep from #1920:
- CustomSignUp/saga.js Url improvements
- deletion of:
  - station_sync/mapping
  - certifier_country Model
  - country Model
  
 Things/references that were not deleted:
- migrations - for obvious reasons
- seed table database - need to review if this affects migrations or testing
- Unused Tables in the database - Will make a jira ticket to review this
- Other unused files/features discovered along the way - Will make another Jira ticket for this as well

Using `git ls-files | xargs wc -l` we see there are about ~501,783 lines of code so this is a 3% of that lol.

Jira link: [LF-3271](https://lite-farm.atlassian.net/browse/LF-3271)

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [ ] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain):

There are a few areas I would expect to see issues if they were being caused:
- Biodiversity insights
- Finances: labour expenses (there are existing console errors here with the dropdown)
- Authentication: (shift and shift task was the most intertwined but still unused)

Jest tests:
- still are working for me with less tests skipped
- some removed tests have been disabled for 2+ years

Otherwise I have no way to test this other than "time will tell" and it looks good to me?

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

[LF-3271]: https://lite-farm.atlassian.net/browse/LF-3271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Duncan-Brain** [comment]
@iperdomo thanks! I botched my merge with integration by forgetting to pull latest changes. Woops!

**Review by iperdomo — APPROVED** [review]
> "Deleted code is debugged code." - Jeff Sickel

**Comment by Duncan-Brain** [comment]
And the two new tickets created from this PR:

[LF-3356 - Decide what to do with unused tables](https://lite-farm.atlassian.net/browse/LF-3356)

[LF-3355 - Continue Deleting Unused Code](https://lite-farm.atlassian.net/browse/LF-3355)

File: packages/webapp/src/apiConfig.js [hunk 1]
```diff
@@ -37,33 +37,31 @@ export const userUrl = `${URI}/user`;
 export const pseudoUserUrl = `${URI}/user/pseudo`;
```
Comment (SayakaOno) [inline packages/webapp/src/apiConfig.js:37]: Thank you for working on this! Everything looks good to me.
Were you going to remove comments on this file?

**Review by kathyavini — APPROVED** [review]
Incredible!! It kind of feels like the code version of when you get a massive haircut and then your head feels so light afterwards 😉 

I haven't found any missed dependencies or anything (and I trust QA to for sure dig them up if they're in there!)

Looks great to me!

**Pull request merged**